src/ssl.c

```diff
@@ -6385,7 +6383,7 @@ int ProcessBuffer(WOLFSSL_CTX* ctx, const unsigned char* buff,
         #ifndef NO_RSA
             haveRSA = 1;
         #endif
-        #ifndef NO_CERTS
+        #ifndef NO_CERTS // NOLINT(readability-redundant-preprocessor)
```

Comment:
Does the lint also support C style `/* */`?

Reply:
no :-[

comment from wolfssl-multi-test says it all:
```
    # C++ comments:
    # note the pattern allows //-comments of the form "// NOLINT" or "// cppcheck" -- these suppressions have no /**/ alternatives.
```